apps/prairielearn/src/sync/sharing.ts

```
@@ -226,6 +226,36 @@ export function checkInvalidSharedAssessments(
   return existInvalidSharedAssessment;
 }
 
+export function checkInvalidSharedCourseInstances(
+  courseData: CourseData,
+  logger: ServerJobLogger,
+): boolean {
+  return false;
+  const invalidSharedCourseInstances = new Set<string>();
+
+  for (const courseInstanceKey in courseData.courseInstances) {
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)